src/Microsoft.AspNetCore.Blazor.Browser.JS/package.json

```diff
@@ -13,6 +13,8 @@
     "ts-loader": "^4.4.1",
     "typescript": "^2.9.2",
     "webpack": "^4.12.0",
-    "webpack-cli": "^3.0.8"
+    "webpack-cli": "^3.0.8",
+    "@aspnet/signalr": "^1.0.0",
+    "@aspnet/signalr-protocol-msgpack": "^1.0.0"
```

Comment:
The JS code is unchanged from your version, I didn't have to make any changes here though I expect I will have to integrate the rendererId change

Reply:
Great!